tests/testing.py

```diff
@@ -58,6 +62,7 @@ def setUp(self):
         self.apt_auth_file = Path(self.tempdir.join('auth.conf'))
         self.apt_method_https = self.bin_dir / 'apt-method-https'
         self.ca_certificates = self.bin_dir / 'update-ca-certificates'
+        self.snap = self.bin_dir / 'snap'
```

Comment:
What is this used for? Removing it doesn't make any tests fail.

Reply:
Probably a leftover, I think I had it referenced before, dropping it